### ISSUE TITLE
Anchors functioning as buttons missing button roles

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1129]
Same as https://github.com/omeka/omeka-s/issues/2408: Across both admin and the front-end, anchors are often used as buttons but lack button roles (and are thus communicating to screen readers as links). Either button roles should be added or anchor elements should be replaced with buttons.

**Comment by aviannamiller** [comment]
Compiling instances, feel free to add/edit:
- "On the 'Items' page, each item listed has the name of the item, 'Details', 'Edit', and 'Delete' links. However, the Details and Delete links do not navigate to new pages. Instead, Details expands an inline panel and Delete opens a modal. Because they trigger in-place actions rather than navigation, they should be implemented as buttons instead of links for proper semantics and accessibility.
  
  This also includes the 'Show Details' link on the same page that is present before and after the list of items."
- Appearance > Navigation page, 'reset navigation' button is announced as link
